amundsen_application/static/js/components/common/SearchBar/index.tsx

```diff
@@ -98,8 +98,10 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
   };
 
   componentDidUpdate = (prevProps: SearchBarProps) => {
-    if (this.props.searchTerm !== prevProps.searchTerm) {
-      this.setState({ searchTerm: this.props.searchTerm });
+    const { searchTerm } = this.props;
```

Comment:
Does this behave the same as {searchTerm} = this.props.searchTerm? might just be a gap in my understanding of hoe React works but wanted to ask anyway.

Reply:
Yes, with ES2015 destructuring, we are declaring a new `const` named `searchTerm` that points to the value on this.props.searchTerm.